.travis.yml

```diff
@@ -9,35 +9,51 @@ env:
     - BYOND_MINOR="1418"
     - NODE_VERSION="4"
     - RUST_G_VERSION="0.3.0"
-    - BUILD_TOOLS=false
-    - BUILD_TESTING=false
-    - DM_MAPFILE="loadallmaps"
-    matrix:
-    - BUILD_TOOLS=true
-    - BUILD_TESTING=true
-    - BUILD_TESTING=false
-
-cache:
-  directories:
-    - $HOME/BYOND-${BYOND_MAJOR}.${BYOND_MINOR}
-    - tgui/node_modules
-    - $HOME/.rustup
-    - $HOME/.cargo
-
-
-addons:
-  apt:
-    packages:
-      - libc6-i386
-      - libgcc1:i386
-      - libstdc++6:i386
-      - gcc-multilib
-      - python3
-      - python3-pip
-      - libssl-dev:i386
+matrix:
+  include:
+    - env:
+      - BUILD_TOOLS=true
+      addons:
+        apt:
+          packages:
+            - python3
+            - python3-pip
+      cache:
+        directories:
+          - tgui/node_modules
+    - env:
+      - BUILD_TESTING=true
+      - BUILD_TOOLS=false
+      addons:
+        apt:
+          packages:
+            - libc6-i386
+            - libstdc++6:i386
+      cache:
+        directories:
+          - $HOME/BYOND-${BYOND_MAJOR}.${BYOND_MINOR}
+    - env:
+      - BUILD_TESTING=false
+      - BUILD_TOOLS=false
+      addons:
+        mariadb: '10.2'
```

Comment:
Mariadb 10.3 is the latest stable release, probably better to use that if travis supports it.

Reply:
Readme specifies 10.2 as minimum supported so I don't think we should change that